GUI/MetaDraw.xaml.cs

```diff
@@ -155,7 +156,7 @@ private void DrawPsm(int oneBasedScanNumber, string fullSequence = null)
             mainViewModel.DrawPeptideSpectralMatch(msDataScanToDraw, psmToDraw);
 
             // draw annotated base sequence
-            DrawAnnotatedBaseSequence(psmToDraw);
```

Comment:
is this available for XL yet? maybe check to see if the PSM is a crosslink, and draw it if it isn't?

Reply:
Ready to merge.
The DrawAnnotatedBaseSequence will be changed in the future for glyco or crosslink peptide.